ion/tokenizer.go

```diff
@@ -583,7 +583,7 @@ func (t *tokenizer) readString() (string, error) {
 			return "", err
 		}
 		// -1 denotes EOF, and new lines are not allowed in short string
-		if c == -1 || c == '\n' || isProhibitedControlChar(c) {
+		if c == -1 || c == '\n' || isProhibitedControlChar(c) || isClobWithInvalidChar(clob, c) {
```

Comment:
`... || (clob && !isASCII(c))` would read more naturally to me here 🤷.

Reply:
> `... || (clob && !isASCII(c))` would read more naturally to me here 🤷.

This would work for me too.

Reply:
Will change.